### PR TITLE
Fix/aact 166 readme updates

### PR DESCRIPTION
**Pull request by crystal-wb**
- I setup aact-admin from scratch and fixed the readme to match the steps I took
- since the .example.env was not being used I removed it
- since the setup files are not need I removed them
- the set_up method was not working so I removed it, opting instead for manual setup of databases

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,60 +3,80 @@ Administer AACT: Aggregated Analysis of ClinicalTrials.gov
...
+- `template1=# \q` (quit out of postgres)  
+
+- Clone the aact-admin repo.  
+
+- `cd` into the aact-admin directory  
+
+- run `bundle install` (bundle version should be '~> 1.17.3')  
+
+if you run into issues with bundle installing `libv8` and the `rubyracer` on a mac you can follow these steps
+`brew install v8-315`
```
Comment (bykasya) [inline README.md:36]: Could you add <br> after "these steps"  so it would look like a list?

**Review by bykasya — APPROVED** [review]
LGTM! Thank you for these changes, the direction is much more clear now! I just requested small styling changes.

File: README.md (hunk 1 above)
Comment (Javier-Jimenez-18) [reply README.md:36]: Looks good. Good job.

**Review by Javier-Jimenez-18 — COMMENTED** [review]
Looks good. Great job.

**Review by micronix — APPROVED** [review]
LG

**Pull request merged**